Libs/vtkITK/vtkITKLabelShapeStatistics.h

```diff
@@ -0,0 +1,103 @@
+/*=========================================================================
+
+  Copyright Brigham and Women's Hospital (BWH) All Rights Reserved.
+
+  See COPYRIGHT.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+==========================================================================*/
+
+#ifndef __vtkITKLabelShapeStatistics_h
+#define __vtkITKLabelShapeStatistics_h
+
+#include "vtkITK.h"
+#include "vtkSimpleImageToImageFilter.h"
+
+// VTK includes
+#include <vtkMatrix4x4.h>
+#include <vtkSmartPointer.h>
+#include <vtkVector.h>
+
+// std includes
+#include <map>
+
+class vtkPoints;
+
+/// \brief ITK-based utilities for calculating label statistics.
```

Comment:
Please make the description a bit more detailed and add link to corresponding ITK class where meaning of input flags and computed values are defined.